dcc-business-rules-utils/src/test/test-dcc-validator.ts

```diff
@@ -101,7 +110,13 @@ describe("check data accesses against CertificateType", () => {
         isTrue(null === validationResult.affectedFields)
     })
 
-    // TODO  check regular validations
+    it("works for Vaccination-type rule", () => {
+        const validationResult = validateRule(ruleWithLogicAndFields(if_(var_("payload.v.0.dt"), true, false), [ "v.0.dt" ]))
+        deepEqual(validationResult.metaDataErrors, [])
+        isTrue(null === validationResult.affectedFields)
+    })
+
+    // TODO  test other certificate types?
```

Comment:
probably a good idea :-)

Reply:
Refactored to actually write those tests